pkg/config/template.go

```diff
@@ -53,7 +54,9 @@ func crioTemplateString(group templateGroup, prefix string, displayAll bool, cri
 
 	for _, configItem := range crioTemplateConfig {
 		if group == configItem.group {
-			if displayAll || !configItem.isDefaultValue {
+			if !configItem.isDefaultValue {
+				templateString += strings.Replace(configItem.templateString, "{{ .Comment }} ", "", 1)
```

Comment:
I'm not seeing many cases where {{ .Comment }} is followed by a space, we may need to replace this with 
`strings.Replace(configItem.templateString, "{{ .Comment }}", "", 1)`

Reply:
also we need this to be `-1` so it replaces all instances of Comment